ui/src/turing/components/form/standard_ensembler/LinkedExperimentsContextMenu.js

```diff
@@ -0,0 +1,86 @@
+import React from "react";
+
+import {
+  EuiContextMenu,
+  EuiPopover,
+  EuiButtonEmpty,
+  EuiTextColor,
+  EuiIcon
+} from "@elastic/eui";
+
+export const LinkedExperimentsContextMenu = ({
+  item,
+  projectId,
+  routeToExperimentMappings,
+  isButtonPopoverOpen,
+  setIsButtonPopoverOpen,
+  experimentStatus
+}) => {
+  let numRoutes = routeToExperimentMappings[item.id] ? Object.keys(routeToExperimentMappings[item.id][experimentStatus]).length : 0;
```

Comment:
Should this be `numExperiments`? 🤔

Reply:
Wow that's an awesome catch! Yes it's meant to be `numExperiments`; thanks!